src/main/groovy/de/dkfz/roddy/tools/RoddyIOHelperMethods.groovy

```diff
@@ -162,16 +163,17 @@ class RoddyIOHelperMethods {
     }
 
     /**
-     * This method is a replacement for Apache FileUtils.copyDirectory
-     * It uses the local Roddy command set (with e.g. "cp -r" for Bash shells)
-     * It is designed to retain access rights.
+     * This method copies a file and tries to preserve access rights using Java get/set rights methods.
      * @param src
      * @param tgt
      */
-    public static void copyDirectory(String cmd) {
+    public static void copyDirectory(File src, File tgt) {
         try {
-            //String cmd = Roddy.getLocalCommandSet().getCopyDirectoryCommand(src, tgt);
-            ExecutionHelper.executeSingleCommand(cmd);
+            FileUtils.copyDirectory(src, tgt)
+            tgt.setReadable(src.canRead())
+            tgt.setWritable(src.canWrite())
+            tgt.setExecutable(src.canExecute())
+
         } catch (Exception ex) {
             logger.severe(ex.toString())
```

Comment:
Well, I know it is different because of the original non-strict Roddy design: "Run as far as you can". In that case it makes sense, but yes for a more public design we will take out the exception here.